### PR TITLE
Add post about my experiences building a simple node app for raspberry pi.

### PR DESCRIPTION
**Pull request by pingortle**
## To Do
- [x] Update the slug to better match content

### CODE REVIEW

**Comment by Schoonology** [comment]
Since the title has changed, are you planning on changing the file name / slug?

**Comment by pingortle** [comment]
> Since the title has changed, are you planning on changing the file name / slug?

@Schoonology Yes, I am planning to change the slug. Thanks for the reminder!

**Comment by JuanCaicedo** [comment]
Oooo, just saw that we can live preview this at https://testdoubleblog-pr-157.herokuapp.com/ 😀

File: app/posts/2019-01-18-tot-tunes.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+```
+
+With this simple configuration you get a rock solid service that automatically restarts on reboots and crashes. As a bonus, it also has easy access to your application's logs through the `journalctl` interface.
+
+## Mission accomplished?
+
+Hacking on Raspberry Pi projects is fun and a little addictive. But if you're like me, we also aspire to have a life outside of our little IoT overlords, so there's not always time to spend hours learning a new operating system or language. We need to get an idea implemented quickly so we can get on with our lives with just a pinch more JS automation because you can never have too much JS 😉.
+
+Although I chose node partly as a fun challenge, I think it has other advantages for IoT newcomers. For one thing, nodes about everybody knows a little JS, so it's a way for those folks to get productive fast. If you're not a fan of JS, there is a ton of tooling available to help you write TypeScript, coffeescript, and many other languages that compile down to JS. `npm` has thousands of packages so you can build on others' work, and `pkg` allows us to package all of this up as an executable binary with no requisite dependencies to install.
```
Comment (Schoonology) [inline app/posts/2019-01-18-tot-tunes.md]: I think you have an extra word, and mean:

> For one thing, ~nodes~ about everybody knows a little JS, ...

**Comment by Schoonology** [comment]
This is a lot of fun, and written appropriately casually. I think making editorial changes would result in having to make a _lot_ of editorial changes in order to integrate them, so I'd say _don't_, and "just ship" this piece of simple fun. 😄

**Pull request merged**